index.html

```diff
@@ -234,20 +234,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>20/12/2024</span>
+              <span>17/02/2025</span>
             </div>
             <div class="news-box-content">
-              <p>The release of uyuni-tools for Ubuntu 24.04 was finally successful, we will try to fix the package for Debian 12 as soon as possible.</p>
+              <p>We are having the <a href="https://github.com/uyuni-project/uyuni/discussions/9729" target="_blank">discussion about the new base OS for Uyuni</a> since February 5th, as announced <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/WCJIIEYEKSLOJMJOMG7PEZ65S4ILFPVX/" target="_blank">via email<a> and <a href="https://x.com/UyuniProject/status/1887155791330771145" target="_blank">X</a>, until tomorrow 18th, end of the day. If you still didn't join, leave your comments and feedback, in preparation for a final poll to be scheduled after the discussion is complete.</p>
```

Comment:
```suggestion
              <p>We are having the <a href="https://github.com/uyuni-project/uyuni/discussions/9729" target="_blank">discussion about the new base OS for Uyuni</a> since February 5th, as announced <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/WCJIIEYEKSLOJMJOMG7PEZ65S4ILFPVX/" target="_blank">via email</a> and <a href="https://x.com/UyuniProject/status/1887155791330771145" target="_blank">X</a>, until tomorrow 18th, end of the day. If you still didn't join, leave your comments and feedback, in preparation for a final poll to be scheduled after the discussion is complete.</p>
```